### ISSUE TITLE
Drawing has potential to set negative error bars

### ISSUE THREAD

**Issue by dfujim** [dfujim/bfit#123]
**Describe the bug**

When fitting runs 40706-40708 from 2024

```
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python3.10/tkinter/__init__.py", line 1921, in __call__
    return self.func(*args)
  File "/home/bnmr/py310venv/lib/python3.10/site-packages/bfit/gui/dataline.py", line 93, in <lambda>
    command=lambda:self.draw(figstyle='data'),
  File "/home/bnmr/py310venv/lib/python3.10/site-packages/bfit/gui/dataline.py", line 248, in draw
    data.draw(d, figstyle=figstyle, label=self.label.get(),
  File "/home/bnmr/py310venv/lib/python3.10/site-packages/bfit/backend/fitdata.py", line 734, in draw
    plt.errorbar(figstyle, self.id, x, ac, dac, label=label, **drawargs)
  File "/home/bnmr/py310venv/lib/python3.10/site-packages/bfit/backend/PltTracker.py", line 266, in errorbar
    obj = ax.errorbar(x, y, yerr=yerr, xerr=xerr, fmt=fmt, ecolor=ecolor,
  File "/home/bnmr/py310venv/lib/python3.10/site-packages/matplotlib/__init__.py", line 1473, in inner
    return func(
  File "/home/bnmr/py310venv/lib/python3.10/site-packages/matplotlib/axes/_axes.py", line 3743, in errorbar
    raise ValueError(
ValueError: 'yerr' must not contain negative values
```

**To Reproduce**
Steps to reproduce the behaviour:
1. Load runs 40706-40708 from 2024 and fit with biexp
2. Shared fraction
3. See error

**Desktop (please complete the following information):**
- isdaq01

**Comment by dfujim** [comment]
Was on combined normalized mode

**Comment by dfujim** [comment]
Fixed in a06db1a05bfe6f35c5d225fcb9792e478e23a459
